src/util/winwrap.cpp

```diff
@@ -602,14 +602,24 @@ namespace vind
             // include the null terminator
             auto size = static_cast<DWORD>(str.size() * sizeof(wchar_t) + 1) ;
 
-            auto result = RegGetValueW(
+            const std::wstring version_keys[2] = {L"DisplayVersion", L"CurrentVersion"};
```

Comment:
RegGetValueW takes `LPCWSTR` type, so should use one of them.

1. Type definition.
```cpp
const auto version_keys[] = 
```

2. Reference in `RegGetValueW`.
```cpp
version_key.c_str()
```